packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -1486,6 +1488,11 @@ export class AppStudioPluginImpl {
 
     let manifest = (await fs.readFile(await this.getManifestTemplatePath(ctx.root))).toString();
 
+    if (isMultiEnvEnabled()) {
```

Comment:
Before multi-env is enabled, will the new manifest template be rendered correctly?